main/install/css/inputShowPwd.css

```diff
@@ -0,0 +1,58 @@
+* {
+	box-sizing: border-box;
+	font-size: 16px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5720016)